releasenotes/notes/add-dd_call_basic_config-env-15570281d176f438.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add ``DD_CALL_BASIC_CONFIG={true,false}`` environment variable to control whether ``ddtrace`` calls ``logging.basicConfig``.
+    By default when using ``ddtrace-run`` or running in debug mode, we always call ``logging.basicConfig`` to ensure there is always a
+    root handler. This has compatibility issues for some logging configurations. With this new environment variable you can use
+    ``DD_CALL_BASIC_CONFIG=false`` to skip calling ``logging.basicConfig``. The default value is ``true`` to maintain existing
+    behaviors.
```

Comment:
```suggestion
    behavior.
```

Reply:
lol didn't think you'd take it 😆  wonder if the spell checker will fail it

Reply:
probably